packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -230,6 +231,28 @@ describe("Garbage Collection Tests", () => {
 				assert(gc.gcEnabled, "gcEnabled incorrect");
 				assert.equal(gc.latestSummaryGCVersion, 1, "latestSummaryGCVersion incorrect");
 			});
+			it("gcEnforcementMinCreateContainerRuntimeVersion later than persisted value, sweepEnabled false", () => {
```

Comment:
This illustrates how GC enforcement would be disabled for an old document.  It's using the `sweepEnabled` setting but would also apply to tombstone failure.